tools/windows/install-help/cal/customactiondata.cpp

```diff
@@ -319,6 +336,22 @@ bool CustomActionData::parseUsernameData()
             _ddUserExists = true;
             _sid = std::move(sidResult.Sid);
 
+            if (_logonCli != nullptr)
+            {
+                BOOL isServiceAccount;
+                DWORD result = _logonCli->NetIsServiceAccount(
```

Comment:
Technically NetIsServiceAccount return NTSTATUS and should be checked for STATUS_SUCCESS even though physically it is the same thing but NTSTATUS is signed and DWORD is not

Reply:
Yep, I know about this conflict. Sometime I copy past NTSTAUS definition and sometime include <winternl.h>. Try the later one it may work. But otherwise it is not a big deal.

Reply:
What includes ntsecapi? Is it needed?